### ISSUE TITLE
Problem: Remove array duplicates (in-place)

### ISSUE THREAD

**Issue by ShambhaviSharma0110** [MakeContributions/DSA#294]
I would like to submit my solution to the following question in c++;

Given a sorted array nums, remove the duplicates in-place such that each element appears only once and returns the new length.

Do not allocate extra space for another array, you must do this by modifying the input array in-place with O(1) extra memory.

**Comment by ming-tsai** [comment]
Hi @ShambhaviSharma0110,
I will wait for your pull request

**Comment by Arsenic-ATG** [comment]
This issue is now open for all to work upon.

If someone want to work on this then feel free to notify the members here to get assigned or you can also open a PR with the algorithm for reviewing.

**Comment by JesusFidel** [comment]
Would be happy to take it. Can you please assign me? thanks.

**Comment by chairmannone** [comment]
Can I try as first issue?

**Comment by Arsenic-ATG** [comment]
@chairmannone, currently @JesusFidel must be working on it in C++. 

If you also want to work on this then you can either pick another language to implement it or discuss with @JesusFidel in case you both want to work on this as a team ( although I don't think this issue is big enough to require 2 programs but we have no problem with it ).

**Comment by JesusFidel** [comment]
@chairmannone you can work on it. I am going to take a while till finalize it because these days are being busy to me. Take this issue if you like and I can take care of another issue in the future. Or, implement this algorithm in C instead of C++ would add value as well.

**Comment by i-am-santosh17** [comment]
Hello, I would love to take this, Please assign this to me

**Comment by i-am-santosh17** [comment]
can you tell me, what is the expected input and output of the function? I mean Do I have to return an output array or just the length of array?

**Comment by Arsenic-ATG** [comment]
> can you tell me, what is the expected input and output of the function? I mean Do I have to return an output array or just the length of array?

@i-am-santosh17 This is not a competitive programming platform, so there are little to no restrictions here and most of the decisions are up to you to take on your program. Just make sure that It should works correctly for any general used case. 🙂

Also you can always add more functionality to your program in future.

**Comment by i-am-santosh17** [comment]
> > can you tell me, what is the expected input and output of the function? I mean Do I have to return an output array or just the length of array?
> 
> @i-am-santosh17 This is not a competitive programming platform, so there are little to no restrictions here and most of the decisions are up to you to take on your program. Just make sure that It should works correctly for any general used case. 🙂
> 
> Also you can always add more functionality to your program in future.

sure, thanks

**Comment by i-am-santosh17** [comment]
> > > can you tell me, what is the expected input and output of the function? I mean Do I have to return an output array or just the length of array?
> > 
> > 
> > @i-am-santosh17 This is not a competitive programming platform, so there are little to no restrictions here and most of the decisions are up to you to take on your program. Just make sure that It should works correctly for any general used case. 🙂
> > Also you can always add more functionality to your program in future.
> 
> sure, thanks

I have created pull request, for this. If you can if everything is alright?
